.travis.yml

```diff
@@ -92,151 +63,149 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X.
+
+      # Try MacOS X.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="scikit-image jplephem $ASDF_PIP_DEP"
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
+               CONDA_DEPENDENCIES="clang"
                CCOMPILER=clang
                EVENT_TYPE='cron'
-               DOUBLE_PLAY=True
+               # DOUBLE_PLAY=True
```

Comment:
Still work in progress for `DOUBLE_PLAY`?

Reply:
Fixed in https://github.com/astropy/astropy/pull/9805 and am now rebasing this